src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -471,10 +467,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
         End Function
 
         Private Function LengthOfLineBreak(StartCharacter As Char, Optional here As Integer = 0) As Integer
-            Debug.Assert(CanGet(here))
+            Dim c As Char
+            Dim res = TryPeek(here,c)
+            Debug.Assert(res)
             Debug.Assert(IsNewLine(StartCharacter))
 
-            Debug.Assert(StartCharacter = Peek(here))
+            Debug.Assert(StartCharacter = c)
```

Comment:
It looks like new code is added for the purpose of the asserts. Consider surrounding the new variable declarations and the dependent asserts with #If DEBUG #End If